src/Features/Core/SolutionCrawler/WorkCoordinator.SemanticChangeProcessor.cs

```diff
@@ -403,7 +403,7 @@ protected override Task ExecuteAsync()
                         }
 
                         data.AsyncToken.Dispose();
-                        return Task.FromResult(true);
+                        return SpecializedTasks.EmptyTask;
```

Comment:
thank you for doing this. didn't know that I returned Task.FromResult ... rather than SpecializedTasks...